src/Workspaces/Core/Portable/Rename/ConflictEngine/MutableConflictResolution.cs

```diff
@@ -89,7 +89,7 @@ internal void UpdateCurrentSolution(Solution solution)
                     var document = CurrentSolution.GetDocument(documentId);
                     var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
 
-                    // For the computeReplacementToken and computeReplacementNode functions, use 
+                    // For the computeReplacementToken and computeReplacementNode functions, use
```

Comment:
Whitespace only change, please revert

Reply:
Looks like one of my plugin is tyring to do that. :) Will reset all the white space change